tests/unit/test_vedabase.py

```diff
@@ -4,17 +4,97 @@
 os.environ["SANDMAN_API"] = "https://veda-api-development.geobigdata.io"
 
 from pyveda import VedaCollection
-from pyveda.db import VedaBase
+from pyveda.vedaset import VedaBase
+from pyveda.fetch.handlers import ClassificationHandler, SegmentationHandler, ObjDetectionHandler
 import json
 from shapely.geometry import shape, box
 from shapely.geometry.polygon import Polygon
+from auth_mock import gbdx
+import vcr
 import unittest
+from unittest import skip
 
+def force(r1, r2):
+    return True
+
+my_vcr = vcr.VCR()
+my_vcr.register_matcher('force', force)
+my_vcr.match_on = ['force']
+
+class VedaBaseLabelTest(unittest.TestCase):
```

Comment:
These label payload handlers are used for both data accessor classes, not just VedaBase. Could be reasonable to move these tests into their own test module specifically called like VedaHandlerTests or whatever since they're so important to downstream objects